{
  "uid" : "7dde718436b2b4f8",
  "name" : "read sensor",
  "historyId" : "a7e15f762ce2f3d55f0e07b1a16707d0",
  "time" : {
    "start" : 1633198967937,
    "stop" : 1633198988547,
    "duration" : 20610
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: No speech contained the expected contentMycroft responded with:\nMycroft: (HomeAssistantSkill)\n\"An error occurred while processing a request in Home Assistant Skill\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'J' as in Juliet. 'P' as in Paul. the number Three. 'C' as in Charlie. 'A' as in Apple. 'K' as in Kilogram.\"\n\n",
  "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 227, in then_contains\n    assert passed, assert_msg\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: No speech contained the expected contentMycroft responded with:\nMycroft: (HomeAssistantSkill)\n\"An error occurred while processing a request in Home Assistant Skill\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'J' as in Juliet. 'P' as in Paul. the number Three. 'C' as in Charlie. 'A' as in Apple. 'K' as in Kilogram.\"\n\n",
    "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 227, in then_contains\n    assert passed, assert_msg\n",
    "steps" : [ {
      "name" : "Given an English speaking user",
      "time" : {
        "start" : 1633198967939,
        "stop" : 1633198967939,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the user says \"give me the value of Mycroft sensor please\"",
      "time" : {
        "start" : 1633198967939,
        "stop" : 1633198967939,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then mycroft reply should contain \"122\"",
      "time" : {
        "start" : 1633198967940,
        "stop" : 1633198977941,
        "duration" : 10001
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: No speech contained the expected contentMycroft responded with:\nMycroft: create.account.dialog(PairingSkill)\n\"Before continuing, please go to account dot mycroft dot A I to create a Mycroft account.\"\nMycroft: pairing.intro.dialog(PairingSkill)\n\"I'm connected to the internet and need to be activated. Open your browser and visit account dot mycroft dot A I slash pair to register this device.\"\nMycroft: (HomeAssistantSkill)\n\"An error occurred while processing a request in Home Assistant Skill\"\n\n",
      "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 227, in then_contains\n    assert passed, assert_msg\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Given an English speaking user",
      "time" : {
        "start" : 1633198978243,
        "stop" : 1633198978243,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the user says \"give me the value of Mycroft sensor please\"",
      "time" : {
        "start" : 1633198978243,
        "stop" : 1633198978244,
        "duration" : 1
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then mycroft reply should contain \"122\"",
      "time" : {
        "start" : 1633198978244,
        "stop" : 1633198988245,
        "duration" : 10001
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: No speech contained the expected contentMycroft responded with:\nMycroft: (HomeAssistantSkill)\n\"An error occurred while processing a request in Home Assistant Skill\"\nMycroft: pairing.code.dialog(PairingSkill)\n\"Enter the code. 'J' as in Juliet. 'P' as in Paul. the number Three. 'C' as in Charlie. 'A' as in Apple. 'K' as in Kilogram.\"\n\n",
      "statusTrace" : "  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/model.py\", line 1806, in run\n    match.run(runner.context)\n  File \"/opt/mycroft-core/.venv/lib/python3.7/site-packages/behave/matchers.py\", line 97, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/utterance_responses.py\", line 227, in then_contains\n    assert passed, assert_msg\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 6,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "sensor"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "7dde718436b2b4f8.json",
  "parameterValues" : [ ]
}